addons/spreadsheet_account/models/account.py

```diff
@@ -46,10 +51,16 @@ def _build_spreadsheet_formula_domain(self, formula_params):
         codes = [code for code in formula_params["codes"] if code]
         if not codes:
             return expression.FALSE_DOMAIN
+
+        account_ids = self._search_accounts_with_codes_or_types(codes=codes).ids
+
         company_id = formula_params["company_id"] or self.env.company.id
         company = self.env["res.company"].browse(company_id)
-        start, end = self._get_date_period_boundaries(
-            formula_params["date_range"], company
+        start, _end = self._get_date_period_boundaries(
+            formula_params["date_from"], company
+        )
+        _start, end = self._get_date_period_boundaries(
+            formula_params["date_to"], company
```

Comment:
I would have expected not specifying `date_to` to not give a boundary instead of taking the one from `date_from` :thinking: 
Might be wrong though

Reply:
Would it not make more sense that the formula ODOO.BALANCE("114", "01/2024",,,,TRUE) returns the results for January 2024 no matter when you are ?